test.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
```

Comment:
We're now sharing these scripts among multiple projects. I wonder if it's a good idea that we make a separate project for them and git submodule them in for use by each project. Then our `test.sh` can just import this file and set a few variables.